js/scenarios.js

```diff
@@ -129,7 +129,8 @@ var Scenarios = [
       {x: 1, y: 2},
       {x: 7, y: 0},
       {x: 7, y: 2},
-      {x: 7, y: 1}
+      {x: 7, y: 1},
+      {x:8, y:1}
```

Comment:
can you correct this line?
{x: 8, y: 1 }
😄

Reply:
OK, added! But note that the spacing is inconsistent throughout the file...

Reply:
yeah, you are right, now I'm noticing it. I will clean-up  and push the changes.
Thanks